test/e2e/tuning_options_test.go

```diff
@@ -320,3 +327,195 @@ func TestHAProxyTimeoutsRejection(t *testing.T) {
 		}
 	}
 }
+
+// TestCookieLen verifies that when logging is enabled and a cookie capture is specified, any captured cookies are
+// truncated at the expected max length.
+func TestCookieLen(t *testing.T) {
+	testCases := []struct {
+		name      string
+		maxLength int
+	}{
+		{"63 bytes", 63},
+		{"64 bytes", 64},
+		{"128 bytes", 128},
+		{"1024 bytes", 1024},
+	}
+	for _, testCase := range testCases {
+		t.Run(testCase.name, func(t *testing.T) {
+			maxLength := testCase.maxLength
+			t.Parallel()
```

Comment:
Out of consistency please can we move this to the first line in the subtest (i.e., before `maxLength`).